pgjdbc/src/test/java/org/postgresql/test/jdbc2/CallableStmtTest.java

```diff
@@ -188,6 +216,84 @@ public void testGetNumericWithoutArg() throws Throwable {
   }
 
   @Test
+  public void testGetTimestampWithoutTimeZoneWithoutArg() throws SQLException {
+    assumeCallableStatementsSupported();
+    CallableStatement call = con.prepareCall(func + pkgName + "getTimestampWithoutTimeZoneWithoutArg () }");
+    try {
+      call.registerOutParameter(1, Types.TIMESTAMP);
+      call.execute();
+      assertEquals(Timestamp.valueOf("2004-10-19 10:23:54.000123"), call.getTimestamp(1));
+    } finally {
+      call.close();
+    }
+  }
+
+  @Test
+  public void testGetTimestampWithoutTimeZoneWithoutArgCalendar() throws SQLException {
+    assumeCallableStatementsSupported();
+    CallableStatement call = con.prepareCall(func + pkgName + "getTimestampWithoutTimeZoneWithoutArg () }");
+    try {
+      call.registerOutParameter(1, Types.TIMESTAMP);
+      call.execute();
+      assertEquals(Timestamp.valueOf("2004-10-19 10:23:54.000123"), call.getTimestamp(1, null));
```

Comment:
Those tests do look like a good case for `@Parameterized`. Have you considered using that kind of test?